subliminal/providers/legendastv.py

```diff
@@ -266,12 +275,19 @@ def get_archives(self, title_id, language_code):
                                             'pack' in archive_soup['class'], 'destaque' in archive_soup['class'],
                                             self.server_url + archive_soup.a['href'][1:])
 
+                data_text = archive_soup.find('p', class_='data').text
+
                 # match rating
-                match = rating_re.match(archive_soup.find('p', class_='data').text)
+                match = rating_re.match(data_text)
                 archive.downloads = int(match.group('downloads'))
                 if match.group('rating') is not None:
                     archive.rating = int(match.group('rating'))
 
+                # timestamp
+                match = timestamp_re.search(data_text)
+                if match:
+                    archive.timestamp = datetime.datetime.strptime(match.group('timestamp'), '%d/%m/%Y - %H:%M')
```

Comment:
I will change that part to something else. The cache should store the tuple `(timestamp, releases)` so that when we get from the cache we can compare timestamp and decide to use it or not.